o2hltcatracking.sh

```diff
@@ -0,0 +1,35 @@
+package: O2HLTCATracking
+version: "%(tag_basename)s"
+tag: hlt_o2_ca_tracking-v1.1
+source: https://github.com/davidrohr/AliRoot
```

Comment:
I think it's not acceptable to make the project depending on another fork of AliRoot

Reply:
Yes, agreed.

Reply:
You are right - it was meant to be temporary, as a matter of fact, until it's included in upstream AliRoot. Unfortunately as you know it has to be used both in AliRoot and O2.

A mitigation could be working on a branch of _our_ AliRoot fork to prevent aliBuild to create yet another full clone of AliRoot.

Also, from what I understand: one of my suggestions was to have a separate default for that, but this part is really needed everywhere.

If @davidrohr agrees (but I sense he will not!) an educated solution would be... to duplicate the code.